src/main/java/com/courtcanva/ccfranchise/dtos/OpenOrderResponseDto.java

```diff
@@ -0,0 +1,42 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OpenOrderResponseDto {
```

Comment:
I think this file should be called OpenOrderGetDto to keep consistency with other dto files